src/crimson/os/seastore/segment_cleaner.cc

```diff
@@ -345,6 +345,7 @@ SegmentCleaner::gc_reclaim_space_ret SegmentCleaner::gc_reclaim_space()
 	      ).safe_then([this, &t] {
 		if (scan_cursor->is_complete()) {
 		  t->mark_segment_to_release(scan_cursor->get_offset().segment);
+                  sc_perf.inc(ss_sm_segments_released);
```

Comment:
There's a corresponding bit in TransactionManager::submit_transaction_direct where we actually do the release -- I think ss_sm_segment_released should be there.

Reply:
@athanatos since sc_perf is defined private, can't  be called in transaction manager function. so I move sc_perf.inc(ss_sm_segments_released)  to segment_cleaner->mark_segment_released  function. Is it OK?